js/script.js

```diff
@@ -0,0 +1,225 @@
+                    </div>
+                </div>
+            </div>
+        </div>
+      
+      
+        <div class="desktop">
+            <!-- ─── Desktop Works ───────────────────── -->
+      
+            <div class="works-container ">
+                <div class="project-content">
+                    <div class="project-title">
+                    <h2>${data[i].title}"</h2>
+                    </div>
+                    <div class="outline">
+                        <ul>
+                        <li class="banner-list">${data[i].outline[0]}"</li>
+                        <span class="dot"></span>
+                        <li class="banner-list dev">${data[i].outline[1]}"</li>
+                        <span class="dot"></span>
+                        <li class="banner-list">${data[i].outline[2]}"</li>
+                        </ul>
+                    </div>
+                    <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                    <div class="tags">
+                        <ul>
+                        <li class="skill">${data[i].tech[0]}"</li>
+                        <li class="skill">${data[i].tech[1]}"</li>
+                        <li class="skill">${data[i].tech[2]}"</li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button type="button" data-id= "${data[i].id}"  class="popBtn btn">See Project</button>
+                    </div>
+                </div>
+                <div class="banner">
+                <img src="${data[i].imageSrc}" class="project-img" alt="project image">
+                </div>
+      
+            </div>
+      
+        </div>
+      </div>
+        `);
+  }
+
+  // ─── Display Popup ───────────────────────────────────────────────────────────
+
+  const btnPop = document.querySelectorAll('.popBtn');
+
+  function showPopUP(currentBtn) {
+    const dataId = +currentBtn;
+    const projectID = data[dataId - 1];
+    popUp.classList.add('popup-show');
+    popUp.insertAdjacentHTML('beforeend', `
+        <div class="works-container">
+        <div class="popup-header">
+            <div class="project-title">
+                <h2>${projectID.title}</h2>
+            </div>
+            <span class="closePopup" id="closePopup">&times;</span>
+        </div>
+        <div class="outline">
+            <ul>
+                <li class="banner-list">${projectID.outline[0]}</li>
+                <span class="dot"> </span>
+                <li class="banner-list dev">${projectID.outline[1]}</li>
+                <span class="dot"></span>
+                <li class="banner-list">${projectID.outline[2]}</li>
+            </ul>
+        </div>
+        <div class="popup-body">
+            <div class="banner">
+                <img src=".${projectID.imageSrc}" class="project-img" alt="project image">
```

Comment:
:red_circle: You are almost there. There is only one problem that has been introduced and it is breaking the image path. Kindly make sure you remove the dot `.` symbol before the dollar sign `$` to point to the root folder.  You can copy and paste this one :arrow_down: 

`<img src="${projectID.imageSrc}" class="project-img" alt="project image">`